charts/kcp/values.yaml

```
@@ -20,6 +20,10 @@ etcd:
   podDisruptionBudget:
     enabled: false
     maxUnavailable: 1
+  # Uncomment the below line in order to specify the name of the storageClass
+  # that the etcd PVC should target. If left commented out, the storageClassName
+  # will be left unset and the default for the cluster will be used.
+  # storageClassName: standard
```

If you set this to an empty string it will not make it into the `{{- if .Values.etcd.storageClassName }}`. I think this is a more common approach to setting these values that I have seen in other charts.

It's also what is done for [volumeClassName](https://github.com/kcp-dev/helm-charts/blob/main/charts/kcp/values.yaml#L44)

```suggestion
  storageClassName: ""
```

---

updated